setup.py

```diff
@@ -10,13 +10,8 @@
     from distutils.core import setup
 
 readme_file = path.join(path.dirname(path.abspath(__file__)), "README.md")
-try:
-    from m2r2 import parse_from_file
-
-    readme = parse_from_file(readme_file)
-except ImportError:
```

Comment:
I'd prefer something like:

`+except (ImportError, DistributionNotFound):`

with the matching import from `pkg_resources`. This way, it's less of a change, if that makes sense.

Reply:
I don't think it makes sense that the package have different description based on your build environment. Maybe we should take the step to declarative style using `setup.cfg` instead?